soleBasket/src/pages/CartPage/CartPage.jsx

```diff
@@ -0,0 +1,75 @@
+import "../CartPage/cartpage.css"
+import {useData} from "../../context/dataContext"
+import {CartProductCard} from "../CartPage/components/CartProductCard"
+import {useEffect, useState} from "react"
+
+export const CartPage = () =>{
+    const [showCartItem, setShowCartItem] = useState(false)
+    const {state} = useData()
+    const {cartlist} = state;
+
+    useEffect(()=>{
+        (cartlist.length!==0) ? setShowCartItem(true):setShowCartItem(false)
+    })
+
+    const cartQty = cartlist.reduce((acc, curr)=> curr = Number(curr.qty)+Number(acc), 0)
+    const cartPrice = cartlist.reduce((acc, curr)=> curr = Number(curr.price)+ Number(acc), 0)
+
+    const discount = 999
```

Comment:
use these values as variables rather than hard coded values

Reply:
I did that since I didnt had discount and delivery charges values in my DB but fair point will keep that in mind!
thank you for the review!